src/libs/actions/Report.js

```diff
@@ -1580,6 +1580,63 @@ Onyx.connect({
     },
 });
 
+/**
+ * Creates an optimistic IOU reportAction
+ *
+ * @param {String} type IOUReportAction type. Can be oneOf(create, decline, cancel, pay).
+ * @param {Number} amount IOU amount in cents.
+ * @param {String} comment user comment for the IOU.
```

Comment:
> user comment for the IOU.
should be
> comment user comment for the IOU. 
OR 
> user: comment for the IOU. 


I guess that you were trying to first name the param before giving the description, but it's not very obvious with the current formatting.

Reply:
Added a hyphen in between